Source/Core/Core/PowerPC/JitInterface.h

```diff
@@ -82,8 +82,11 @@ class JitInterface
   void InvalidateICache(u32 address, u32 size, bool forced);
   void InvalidateICacheLine(u32 address);
   void InvalidateICacheLines(u32 address, u32 count);
-  static void InvalidateICacheLineFromJIT(u32 address, u32 dummy, JitInterface& jit_interface);
-  static void InvalidateICacheLinesFromJIT(u32 address, u32 count, JitInterface& jit_interface);
+  static void InvalidateICacheLineFromJit64(JitInterface& jit_interface, u32 address);
```

Comment:
Why do we need to duplicate these instead of keeping the argument order consistent?

Reply:
Mostly because I'm afraid to touch the JitArm64 code.